client/src/utils/graphql.ts

```diff
@@ -19,8 +22,17 @@ export type GraphEndpoint = {
  * @param chainId
  * @returns GraphEndpoint
  */
-const getGraphQLEndpoint = (chainId: number): GraphEndpoint => {
+const getGraphQLEndpoint = (
+  chainId: number,
+  reactEnv?: any // ProcessEnv
```

Comment:
I am adding this so that we can pass `process.env` to this function - reason being that env variables with cra have to start with `REACT_APP_` but only work within React components

Reply:
Ah, so this was the issue previously with the `.env` variables. So as long as they start with `REACT_APP_` they will get picked up 🙏🏼 was this in the React docs?

Reply:
In the CRA docs somewhere